diff --git a/fuse/compat/getopt.h b/fuse/compat/getopt.h
index e50fd96..bafa8e0 100644
--- a/fuse/compat/getopt.h
+++ b/fuse/compat/getopt.h
@@ -138,7 +138,7 @@ struct option
    `getopt'.  */
 
 #if (defined __STDC__ && __STDC__) || defined __cplusplus
-# if defined(__GNU_LIBRARY__) || defined(VITA) || defined(PSP)
+# if defined(__GNU_LIBRARY__) || defined(VITA) || defined(PSP) || defined(__NEWLIB__)
 /* Many other libraries have conflicting prototypes for getopt, with
    differences in the consts, in stdlib.h.  To avoid compilation
    errors, only prototype getopt for the GNU C library.  */
